variables.tf

```diff
@@ -66,7 +66,12 @@ variable "engine" {
   description = "Engine name, defaults to aurora."
 }
 
-variable engine_version {
+variable "engine_version" {
```

Comment:
Are the double quotes now necessary?

Reply:
Not necessary, I just ran `terraform format` and it added them.

Reply:
@kennethjor it shouldn't do that, what version of terraform are you on?

Reply:
apparently this is intended. it doesn't do it for me. weird https://github.com/hashicorp/terraform/issues/27102